Source/WebCore/accessibility/AccessibilityARIAGridRow.cpp

```diff
@@ -71,7 +71,7 @@ AXCoreObject::AccessibilityChildrenVector AccessibilityARIAGridRow::disclosedRow
     // The contiguous disclosed rows will be the rows in the table that 
     // have an aria-level of plus 1 from this row.
     AccessibilityObject* parent = parentObjectUnignored();
-    if (!is<AccessibilityTable>(*parent) || !downcast<AccessibilityTable>(*parent).isExposable())
+    if (auto* axTable = dynamicDowncast<AccessibilityTable>(parent); !axTable || !axTable->isExposable())
```

Comment:
Should be `*parent` to avoid an unnecessary null check, like in the original code.

Reply:
Done.